clang/test/CIR/CodeGen/vbase.cpp

```diff
@@ -19,7 +19,7 @@ void ppp() { B b; }
 // CIR:  cir.global linkonce_odr @_ZTV1B = #cir.vtable<{#cir.const_array<[#cir.ptr<12> : !cir.ptr<!u8i>, #cir.ptr<null> : !cir.ptr<!u8i>, #cir.global_view<@_ZTI1B> : !cir.ptr<!u8i>]> : !cir.array<!cir.ptr<!u8i> x 3>}>
 
 // VTT for B.
-// CIR:  cir.global linkonce_odr @_ZTT1B = #cir.const_array<[#cir.global_view<@_ZTV1B, [#cir.int<0> : !s32i, #cir.int<0> : !s32i, #cir.int<3> : !s32i]> : !cir.ptr<!u8i>]> : !cir.array<!cir.ptr<!u8i> x 1>
+// CIR:  cir.global linkonce_odr @_ZTT1B = #cir.const_array<[#cir.global_view<@_ZTV1B, [0 : index, 0 : index, 3 : index]> : !cir.ptr<!u8i>]> : !cir.array<!cir.ptr<!u8i> x 1>
```

Comment:
Unfortunately changes in this file can't be verified because test is marked as `XFAIL` (and indeed crashes at runtime).

Reply:
Oh shoot, thanks for patching up speculatively. Looks like this could have been added on top of latest rebase, let me track it and look into it.